subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -149,8 +149,14 @@ enum {
 #else /* !CONFIG_BT_MAX_CONN */
 #define BT_CTLR_ADV_ISO_STREAM_HANDLE_BASE 0
 #endif /* !CONFIG_BT_MAX_CONN */
+#define ADV_ISO_CONN2STREAM_HANDLE(conn_handle) \
+	((conn_handle) - (BT_CTLR_ADV_ISO_STREAM_HANDLE_BASE))
+#define ADV_ISO_STREAM2CONN_HANDLE(stream_handle) \
+	((stream_handle) + (BT_CTLR_ADV_ISO_STREAM_HANDLE_BASE))
```

Comment:
I suggest using same naming convention as i`n ull_iso_types.h`, e.g. `LL_CIS_IDX_FROM_HANDLE`(), or at least streamline by changing that and moving to `lll.h`